.readthedocs.yml

```
@@ -20,7 +20,8 @@ build:
     - asdf install uv 0.5.22
     - asdf global uv 0.5.22
     create_environment:
-    - uv venv
+    - uv venv RTD
+    - . RTD/bin/activate
```

Thanks @danielhollas , good to know!
I'd still think this has to be clarified, for easier maintenance in future: 

```suggestion
    # This creates an environment in path '.venv' which will not be activated, but is picked up with the later two commands, automatically: `uv sync` and `uv run`
    - uv venv
```

---

Sure, having a comment is good, this is somewhat subtle. Here's a more concise suggestion:

```suggestion
    # Create a virtual environment in '.venv/' folder
    # which is picked up automatically by `uv sync` and `uv run` commands below
    - uv venv
```